torch/_functorch/functional_call.py

```diff
@@ -77,3 +77,52 @@ def functional_call(
         parameters_and_buffers = {k: v for d in parameter_and_buffer_dicts for k, v in d.items()}
 
     return nn.utils.stateless.functional_call(module, parameters_and_buffers, args, kwargs)
+
+
+@exposed_in("torch.func")
+def stack_ensembled_state(models):
+    """stack_ensembled_state(models) -> params, buffers
```

Comment:
Sorry just to check -- in the comment thread you said you were going with `stack_module_state`, but the API in this PR is `stack_ensembled_state`. Is this intentional?

"stack_ensembled_state" sounds a bit weird to me -- what is an "ensembled state", and what does it mean to "stack" an "ensembled state"? (I can be convinced, this was just my initial reaction)

Reply:
Ahh good catch--this was unintentional. Changed it to `stack_module_state`